### DISCUSSION TITLE
Organization is flagged

### DISCUSSION THREAD

**Discussion by softdevmis** [community/community#28749] [Other Feature Feedback, Questions, & Ideas] [answered]
I don’t understand what happened to my organization Primary-Structures-Corp.
Would you tell me what to do now?
Thank you very much for your support.
I don’t know why my account has been flagged.

**Comment by airtower-luna** [comment, marked as answer]
That's something you'll have to [contact GitHub support](https://support.github.com/request) about, us other users can't help with account details. Good luck!

**Reply by vksarswat** [reply]
flagged issue what i do now
